back-end/src/sciler/handler/handler_test.go

```diff
@@ -257,10 +257,11 @@ func TestSendInstructionDelay(t *testing.T) {
 	})
 	communicatorMock.On("Publish", "display", string(msg), 3)
 	communicatorMock.On("Publish", "display", string(msg2), 3)
-	handler.SendComponentInstruction("display", inst, "")
-	handler.SendComponentInstruction("display", inst2, "5s")
+	go handler.SendComponentInstruction("display", inst, "")
+	go handler.SendComponentInstruction("display", inst2, "1s")
+	time.Sleep(time.Duration(100) * time.Millisecond)
```

Comment:
time.Duration is not needed here

Reply:
replaced all time.duration in the project to the new form